typescript/infra/config/environments/mainnet3/agent.ts

```diff
@@ -138,6 +142,7 @@ export const hyperlaneContextAgentChainConfig: AgentChainConfig<
     bsc: true,
     celo: true,
     cheesechain: true,
+    eclipse: false,
```

Comment:
not critical but there has been confusion in the past why these are set to false, maybe the `// Cannot scrape non-EVM chains` comment is worth it for this and solana